### ISSUE TITLE
Using a different genesis url when deployed 

### ISSUE THREAD

**Issue by Kampe** [simonwep/ocular#48]
### I've found a bug and checked that ...

- [X] ...there are no open or closed issues that are related to my problem
- [X] ...it's definitely a bug and not a feature request

### Description

OCULAR_GENESIS_HOST does not like being set to "/" "" or null

### Environment

OCULAR_GENESIS_HOST="/"

### Expected behaviour

OCULAR_GENESIS_HOST uses the root and not /api

**Comment by simonwep** [comment]
Could you please elaborate what you're trying to achieve with

> OCULAR_GENESIS_HOST="/"

Or it being set to `null`?

**Comment by Kampe** [comment]
The problem is with the default configuration running in anything except docker compose with nginx.

https://github.com/simonwep/genesis/issues/4 is related

**Comment by simonwep** [comment]
Yeah this will require a change in both projects. Not sure about the frontend though as `OCULAR_GENESIS_HOST` is baked into the build, you'd have to build the docker-image yourself or we'd have to add an `nginx` config for the frontend container to make this a runtime option.

**Comment by Kampe** [comment]
Yeah, not sure that's the right environment variable for setting that, am completely open to any and all options would love to just get it working as it seems like a fantastic tool for my family, great work!

**Comment by simonwep** [comment]
Alright, I added the change to `genesis`. The frontend however is built and fix - it will _require_ that, for a deployment outside of docker-compose, `GENESIS_BASE_URL` is set to `/api/`. If you want to change the URL the frontend makes requests to you will have to build the docker-image yourself, which should be pretty straight forward. Basically just changing `OCULAR_GENESIS_HOST` and running `docker build . -t --env-file .env ocular` or something like that.

**Comment by simonwep** [comment]
Closing for now due to inactivity. Please create a new issue if there any more questions or if you believe this is a mistake :)
